app/src/main/java/org/oppia/android/app/settings/profile/DeleteProfileSuccessDialogFragment.kt

```diff
@@ -0,0 +1,44 @@
+package org.oppia.android.app.settings.profile
+
+import android.app.Dialog
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import org.oppia.android.R
+import org.oppia.android.app.administratorcontrols.AdministratorControlsActivity
+import org.oppia.android.app.fragment.FragmentComponentImpl
+import org.oppia.android.app.fragment.InjectableDialogFragment
+
+class DeleteProfileSuccessDialogFragment : InjectableDialogFragment() {
+
+  companion object {
+    const val TAG = "TAG_DELETE_DIALOG_FRAGMENT"
+
+    fun newInstance(): DeleteProfileSuccessDialogFragment {
+      return DeleteProfileSuccessDialogFragment()
+    }
+  }
+
+  override fun onAttach(context: Context) {
+    super.onAttach(context)
+    (fragmentComponent as FragmentComponentImpl).inject(this)
+  }
+
+  override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+    return AlertDialog.Builder(requireContext(), R.style.OppiaAlertDialogTheme)
+      .setMessage(R.string.profile_edit_delete_successful_message)
+      .setPositiveButton(R.string.log_out_dialog_okay_button) { _, _ ->
+        if (requireContext().resources.getBoolean(R.bool.isTablet)) {
+          val intent =
+            Intent(requireContext(), AdministratorControlsActivity::class.java)
+          intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
+          startActivity(intent)
+        } else {
+          val intent = Intent(requireContext(), ProfileListActivity::class.java)
+          intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
+          startActivity(intent)
+        }
+      }.create()
```

Comment:
This dialog can be cancelable outside of the dialog, I think.

https://github.com/oppia/oppia-android/blob/2c6aebd91635ab8dd31488d60aec987c6c37b811/app/src/main/java/org/oppia/android/app/drawer/ExitProfileDialogFragment.kt#L81

Reply:
I don't quite thing canceling on outside touch is the right decision to use here.
User is required to make an action to move to the next screen.

Maybe @BenHenning can give thoughts on this?

Reply:
I think cancelling outside is fine for this dialog since it doesn't actually require a user decision to proceed, it's just a notice that the operation has finished. If the user misses it, nothing technically changes in the app.

Reply:
Okay. I'll include that.

Reply:
Please set the value to true instead of false.

Reply:
Done